app/root.tsx

```diff
@@ -28,22 +29,21 @@ import {
   findBoroughs,
   findCityCouncilDistricts,
   findCommunityDistrictsByBoroughId,
+  findAgencies,
+  FindAgenciesQueryResponse,
 } from "./gen";
 import { FilterMenu } from "./components/FilterMenu";
+import { SearchByAttributeMenu } from "./components/SearchByAttributeMenu";
 import { LoaderFunctionArgs, LinksFunction } from "@remix-run/node";
 import {
   BoroughDropdown,
   DistrictTypeDropdown,
   CommunityDistrictDropdown,
   CityCouncilDistrictDropdown,
+  AgencyDropdown,
 } from "./components/AdminDropdown";
-import {
-  GoToCityCouncilDistrictBtn,
-  GoToDistrictBtn,
-} from "./components/GoToDistrictBtn";
-import { GoToCommunityDistrictBtn } from "./components/GoToDistrictBtn/GoToCommunityDistrictBtn";
```

Comment:
If we're not using these components, we should clean up their definition files as well